source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+
+An automation has three main parts:
+
+- **[Triggers](https://www.home-assistant.io/docs/automation/trigger/)** - something that starts the automation, like the sun going down or a motion sensor noticing movement
+- **[Conditions](https://www.home-assistant.io/docs/automation/condition/)** - let you decide whether to complete the automation. For example: "only if someone is home".
+- **[Action](https://www.home-assistant.io/docs/automation/action/)** - what the automation does, like "turn on lights"
+
+**[Templates](https://www.home-assistant.io/docs/automation/templating/)** let advanced users use variables and logic in their automations
+
+## Events and state
+
+Entities in Home Assistant are controlled by sending **[events](https://www.home-assistant.io/docs/configuration/events/)**.
+
+Home Assistant remembers the **[state](https://www.home-assistant.io/docs/configuration/state_object/)** of each entity. The state means what *mode* or *condition* the entity is in this moment. For example, a light could have two states: ‘on’ and ‘off’, and a garage door could be in one of two states: ‘open’ or ‘closed’.
+
+A **[scene](https://www.home-assistant.io/docs/scene/)** is a list of entities along with the state you want them each to be in. When you trigger that scene, Home Assistant will figure out the right events to get each of those entities into that state.
+
+For example, let’s imagine you have a scene called “welcome home” that says “lights on, door unlocked”. If you turn on that scene, Home Assistant will check if your lights are on and turn them on if they’re off. Then it will check if your door is unlocked, and unlock it if it’s locked.
+
+## More concepts
+
+- **Zones** are geographical areas on earth. You can check if a device tracker is inside a zone as a trigger or condition in an automation. For example, “turn on the lights when Sam’s Phone comes into the ‘home’ zone”.
+
+## Integrations
+
+Home Assistant is set up to work with devices and services from lots of different companies, including voice assistants, smart devices, cameras, and other home automation software. You can make Home Assistant work with these services by installing **[integrations](https://www.home-assistant.io/integrations)**.
+
+Once installed, integrations can make new entities available, add panels to Lovelace, and more.
+
+_Formerly known as components; not to be confused with add-ons._
+
+## The Home Assistant software
+
+So now you know what Home Assistant is. But how do you start using it?
+
+You probably want to [install Home Assistant](https://www.home-assistant.io/getting-started/) using a disk image, which contains everything you need to run Home Assistant and keep it updated.
```

Comment:
Wait? This page is part of the getting started? So, why link to it?

Reply:
If they got to "core concepts" from another place, they may need a link to the installation instructions, which live in the "getting started" root

Reply:
So it isn't getting started tutorial then? I'm confused. Are you aiming for adding something to the getting started guide, or writing something new?